packages/styled-components/src/constructors/test/createGlobalStyle.test.js

```diff
@@ -230,26 +234,15 @@ describe(`createGlobalStyle`, () => {
 
     render(<Comp />);
     const el = document.querySelector('[data-test-el]');
-    expect(getCSS(document).trim()).toMatchInlineSnapshot(`
-      "/* sc-component-id:sc-global-3005254895 */
```

Comment:
all these snapshots basically look at the inserted CSS in non-speedy mode. In canary we never insert any comment markers anymore, so those are gone. The rest of the CSS remains identical